### ISSUE TITLE
About learning_rate Gradually diminishing,is it necessary?

### ISSUE THREAD

**Issue by xinsuinizhuan** [josephjaspers/blackcat_tensors#34]
A yolo's learning_rate, it decrease at epoche's 80% and 90%, whether it is necessary in our model?

**Comment by josephjaspers** [comment]
It is not necessary but it is recommended to decrease the step size in gradient descent. 
(So when the neural network is getting close to the minima it decreases the step-size so as to not jump-over the minima).
